server/migrations/20170724173703-create-document.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Documents', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      title: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        validate: {
+          notEmpty: {
+            args: true,
+            msg: 'Document title cannot be empty'
+          }
+        }
+      },
+      content: {
+        type: Sequelize.STRING,
```

Comment:
`Sequelize.STRING` has a character limit of 255. If you do not intend to limit the content space available to users; use `Sequelize.TEXT`.